### ISSUE TITLE
Help with generate PDF file in Symfony 3.4 with Spipu 5.2.0 in production server

### ISSUE THREAD

**Issue by mugan86** [spipu/html2pdf#369]
Goodnight,
Surprisingly in local and on a server that I had before using this new one I am doing great but since I have had to work with this server and I can not solve it, I am looking for some help, to see if you can guide me.

The "conflicting" URL is this: https://anartzmugika.com/gesti-hotels/web/client/pdf/create that is request to generate simple PDF, in this moment only print "Hello world"...

I define correctly Controller action and pdf generate options:

```
> <?php
> 
> namespace BackendBundle\Controller;
> 
> use AppBundle\Controller\BaseController;
> use Spipu\Html2Pdf\Html2Pdf;
> 
> class DefaultController extends BaseController
> {
>     public function pdfAction()
>     {
> 		$html2pdf = new Html2Pdf();
> 		$html2pdf->writeHTML('<h1>HelloWorld</h1>This is my first test');
> 		$html2pdf->output();
> 
>     }
> }
```
I use too FOS Rest Bundle because I need to create a API about my project and to default define with JSON format (when start pat with ^/api/...)

If use ^/<other>, for example my_localhost/client/pdf/create must be generate a html template but not create!! (only when i want create PDF file, if call to show simple html work OK .---> https://anartzmugika.com/gesti-hotels/web)

In config.yml I define config to manage API (in local 100% work and other production server too)

```
# FOSRest Configuration
fos_rest:
    body_listener: true
    allowed_methods_listener: true
    routing_loader:
       default_format: json
       include_format: false
    format_listener:
        rules:
            - { path: ^/api, priorities: [ 'json' ], fallback_format: json, prefer_extension: true }
            - { path: '^/', priorities: [ 'text/html', '*/*'], fallback_format: html, prefer_extension: true }
    param_fetcher_listener: true
    view:
        view_response_listener: 'force'
        formats:
            json: true
```

If call to address: https://anartzmugika.com/gesti-hotels/web/client/pdf/create (to generate PDF, throw thisexception...I dont understand that passed, i'm very frustrated!)

If used to other server, the result is OK: http://gestihotels.mugan86.com/client/pdf/create

My objective is generate PDF and using same configuration to work correctly.

```
[2018-08-30 20:00:30] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_class" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_class\" service is deprecated since version 5.2 at /usr/home/anartzmugika/www/gesti-hotels/var/cache/prod/ContainerQwuxw6c/getSensioFrameworkExtra_Routing_Loader_AnnotClassService.php:8)"} []
[2018-08-30 20:00:30] php.INFO: User Deprecated: The "Sensio\Bundle\FrameworkExtraBundle\Routing\AnnotatedRouteControllerLoader" class is deprecated since version 5.2. Use "Symfony\Bundle\FrameworkBundle\Routing\AnnotatedRouteControllerLoader" instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"Sensio\\Bundle\\FrameworkExtraBundle\\Routing\\AnnotatedRouteControllerLoader\" class is deprecated since version 5.2. Use \"Symfony\\Bundle\\FrameworkBundle\\Routing\\AnnotatedRouteControllerLoader\" instead. at /usr/home/anartzmugika/www/gesti-hotels/vendor/sensio/framework-extra-bundle/Routing/AnnotatedRouteControllerLoader.php:19)"} []
[2018-08-30 20:00:30] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_dir" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_dir\" service is deprecated since version 5.2 at /usr/home/anartzmugika/www/gesti-hotels/var/cache/prod/ContainerQwuxw6c/getSensioFrameworkExtra_Routing_Loader_AnnotDirService.php:8)"} []
[2018-08-30 20:00:30] php.INFO: User Deprecated: The "sensio_framework_extra.routing.loader.annot_file" service is deprecated since version 5.2 {"exception":"[object] (ErrorException(code: 0): User Deprecated: The \"sensio_framework_extra.routing.loader.annot_file\" service is deprecated since version 5.2 at /usr/home/anartzmugika/www/gesti-hotels/var/cache/prod/ContainerQwuxw6c/getSensioFrameworkExtra_Routing_Loader_AnnotFileService.php:8)"} []
[2018-08-30 20:00:30] php.DEBUG: Warning: Zend OPcache API is restricted by "restrict_api" configuration directive {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/usr/home/anartzmugika/www/gesti-hotels/vendor/symfony/symfony/src/Symfony/Component/Config/ResourceCheckerConfigCache.php\",\"line\":141,\"trace\":[{\"file\":\"/usr/home/anartzmugika/www/gesti-hotels/vendor/symfony/symfony/src/Symfony/Component/Routing/Router.php\",\"line\":301,\"function\":\"write\",\"class\":\"Symfony\\\\Component\\\\Config\\\\ResourceCheckerConfigCache\",\"type\":\"->\"}],\"count\":1})"} []
[2018-08-30 20:00:30] request.INFO: Matched route "client_pdf". {"route":"client_pdf","route_parameters":{"_controller":"BackendBundle\\Controller\\DefaultController::pdfAction","_route":"client_pdf"},"request_uri":"https://anartzmugika.com/gesti-hotels/web/client/pdf/create","method":"GET"} []
[2018-08-30 20:00:30] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\DebugHandlersListener::configure". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\DebugHandlersListener::configure"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\ValidateRequestListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\ValidateRequestListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Nelmio\CorsBundle\EventListener\CorsListener::onKernelRequest". {"event":"kernel.request","listener":"Nelmio\\CorsBundle\\EventListener\\CorsListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\FragmentListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\FragmentListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\FormatListener::onKernelRequest". {"event":"kernel.request","listener":"FOS\\RestBundle\\EventListener\\FormatListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Bundle\FrameworkBundle\EventListener\ResolveControllerNameSubscriber::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Bundle\\FrameworkBundle\\EventListener\\ResolveControllerNameSubscriber::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\BodyListener::onKernelRequest". {"event":"kernel.request","listener":"FOS\\RestBundle\\EventListener\\BodyListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Knp\Bundle\PaginatorBundle\Subscriber\SlidingPaginationSubscriber::onKernelRequest". {"event":"kernel.request","listener":"Knp\\Bundle\\PaginatorBundle\\Subscriber\\SlidingPaginationSubscriber::onKernelRequest"} []
[2018-08-30 20:00:30] php.DEBUG: Warning: Zend OPcache API is restricted by "restrict_api" configuration directive {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/usr/home/anartzmugika/www/gesti-hotels/vendor/symfony/symfony/src/Symfony/Component/Cache/Traits/PhpFilesTrait.php\",\"line\":148,\"trace\":[{\"file\":\"/usr/home/anartzmugika/www/gesti-hotels/vendor/symfony/symfony/src/Symfony/Component/Cache/Adapter/AbstractAdapter.php\",\"line\":234,\"function\":\"doSave\",\"class\":\"Symfony\\\\Component\\\\Cache\\\\Adapter\\\\PhpFilesAdapter\",\"type\":\"->\"}],\"count\":4})"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "FOS\RestBundle\EventListener\ParamFetcherListener::onKernelController". {"event":"kernel.controller","listener":"FOS\\RestBundle\\EventListener\\ParamFetcherListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\ControllerListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\ParamConverterListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\HttpCacheListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\TemplateListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller_arguments" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelControllerArguments". {"event":"kernel.controller_arguments","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\SecurityListener::onKernelControllerArguments"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller_arguments" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\IsGrantedListener::onKernelControllerArguments". {"event":"kernel.controller_arguments","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\IsGrantedListener::onKernelControllerArguments"} []
[2018-08-30 20:00:30] request.CRITICAL: Uncaught PHP Exception LogicException: "An instance of Symfony\Bundle\FrameworkBundle\Templating\EngineInterface must be injected in FOS\RestBundle\View\ViewHandler to render templates." at /usr/home/anartzmugika/www/gesti-hotels/vendor/friendsofsymfony/rest-bundle/View/ViewHandler.php line 346 {"exception":"[object] (LogicException(code: 0): An instance of Symfony\\Bundle\\FrameworkBundle\\Templating\\EngineInterface must be injected in FOS\\RestBundle\\View\\ViewHandler to render templates. at /usr/home/anartzmugika/www/gesti-hotels/vendor/friendsofsymfony/rest-bundle/View/ViewHandler.php:346)"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\DebugHandlersListener::configure". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\DebugHandlersListener::configure"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\ValidateRequestListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\ValidateRequestListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Nelmio\CorsBundle\EventListener\CorsListener::onKernelRequest". {"event":"kernel.request","listener":"Nelmio\\CorsBundle\\EventListener\\CorsListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\FragmentListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\FragmentListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\FormatListener::onKernelRequest". {"event":"kernel.request","listener":"FOS\\RestBundle\\EventListener\\FormatListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Bundle\FrameworkBundle\EventListener\ResolveControllerNameSubscriber::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Bundle\\FrameworkBundle\\EventListener\\ResolveControllerNameSubscriber::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\BodyListener::onKernelRequest". {"event":"kernel.request","listener":"FOS\\RestBundle\\EventListener\\BodyListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::onKernelRequest". {"event":"kernel.request","listener":"Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.request" to listener "Knp\Bundle\PaginatorBundle\Subscriber\SlidingPaginationSubscriber::onKernelRequest". {"event":"kernel.request","listener":"Knp\\Bundle\\PaginatorBundle\\Subscriber\\SlidingPaginationSubscriber::onKernelRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "FOS\RestBundle\EventListener\ParamFetcherListener::onKernelController". {"event":"kernel.controller","listener":"FOS\\RestBundle\\EventListener\\ParamFetcherListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ControllerListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\ControllerListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\ParamConverterListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\ParamConverterListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\HttpCacheListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener::onKernelController". {"event":"kernel.controller","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\TemplateListener::onKernelController"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller_arguments" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\SecurityListener::onKernelControllerArguments". {"event":"kernel.controller_arguments","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\SecurityListener::onKernelControllerArguments"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.controller_arguments" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\IsGrantedListener::onKernelControllerArguments". {"event":"kernel.controller_arguments","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\IsGrantedListener::onKernelControllerArguments"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "FOS\RestBundle\EventListener\AllowedMethodsListener::onKernelResponse". {"event":"kernel.response","listener":"FOS\\RestBundle\\EventListener\\AllowedMethodsListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\ResponseListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\Security\\Http\\RememberMe\\ResponseListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". {"event":"kernel.response","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\HttpCacheListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\Firewall\ContextListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\Security\\Http\\Firewall\\ContextListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SaveSessionListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\StreamedResponseListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::onKernelFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::onKernelFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\Security\Http\Firewall\ExceptionListener::onKernelException". {"event":"kernel.exception","listener":"Symfony\\Component\\Security\\Http\\Firewall\\ExceptionListener::onKernelException"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Bundle\SwiftmailerBundle\EventListener\EmailSenderListener::onException". {"event":"kernel.exception","listener":"Symfony\\Bundle\\SwiftmailerBundle\\EventListener\\EmailSenderListener::onException"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelException". {"event":"kernel.exception","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelException"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.exception" to listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException". {"event":"kernel.exception","listener":"Symfony\\Component\\HttpKernel\\EventListener\\ExceptionListener::onKernelException"} []
[2018-08-30 20:00:30] event.DEBUG: Listener "Symfony\Component\HttpKernel\EventListener\ExceptionListener::onKernelException" stopped propagation of the event "kernel.exception". {"event":"kernel.exception","listener":"Symfony\\Component\\HttpKernel\\EventListener\\ExceptionListener::onKernelException"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "FOS\RestBundle\EventListener\AllowedMethodsListener::onKernelResponse". {"event":"kernel.response","listener":"FOS\\RestBundle\\EventListener\\AllowedMethodsListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\ResponseListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\ResponseListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\Security\Http\RememberMe\ResponseListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\Security\\Http\\RememberMe\\ResponseListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Sensio\Bundle\FrameworkExtraBundle\EventListener\HttpCacheListener::onKernelResponse". {"event":"kernel.response","listener":"Sensio\\Bundle\\FrameworkExtraBundle\\EventListener\\HttpCacheListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\SaveSessionListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SaveSessionListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.response" to listener "Symfony\Component\HttpKernel\EventListener\StreamedResponseListener::onKernelResponse". {"event":"kernel.response","listener":"Symfony\\Component\\HttpKernel\\EventListener\\StreamedResponseListener::onKernelResponse"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.finish_request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::onKernelFinishRequest". {"event":"kernel.finish_request","listener":"Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::onKernelFinishRequest"} []
[2018-08-30 20:00:30] event.DEBUG: Notified event "kernel.terminate" to listener "Symfony\Bundle\SwiftmailerBundle\EventListener\EmailSenderListener::onTerminate". {"event":"kernel.terminate","listener":"Symfony\\Bundle\\SwiftmailerBundle\\EventListener\\EmailSenderListener::onTerminate"} []

```

Same is a complete bullshit but I have lost 3 days and I do not find anything at all, that's why I come here in search of some guidance, in case someone has passed.
Those of the hosting that I have contracted are washing their hands, and they say that it does not work because I have programmed it badly. I imagine that if I had done wrong it would not work either in the local or in the other server, or is it not true?

I thank you in advance to read me and if someone can help me out I'll thank you a lot. Thank you very much and I am sorry to be late and come with these errors.

regards

**Comment by LittleBigFox** [comment]
Hi!
I'm not great with Symfony, but:

`[2018-08-30 20:00:30] request.CRITICAL: Uncaught PHP Exception LogicException: "An instance of Symfony\Bundle\FrameworkBundle\Templating\EngineInterface must be injected in FOS\RestBundle\View\ViewHandler to render templates." at /usr/home/anartzmugika/www/gesti-hotels/vendor/friendsofsymfony/rest-bundle/View/ViewHandler.php line 346 {"exception":"[object] (LogicException(code: 0): An instance of Symfony\\Bundle\\FrameworkBundle\\Templating\\EngineInterface must be injected in FOS\\RestBundle\\View\\ViewHandler to render templates. at /usr/home/anartzmugika/www/gesti-hotels/vendor/friendsofsymfony/rest-bundle/View/ViewHandler.php:346)"} []`

I think it's the only critical you have, so you better ask in the symfony github: (https://github.com/symfony/symfony)

Regards,
LBF

**Comment by mugan86** [comment]
Thank you very much friend!
The surprise of all this is that I have tried a free server (webhost) and it works perfectly. I'll ask you where you sent me, I'll close the "issue" because I do not consider it that way, in that case it would be something Symfony that should send it to the other repository as you have indicated.

This is the example I've tested on webhost: https://gesti-hotels-app.000webhostapp.com/web/app.php/client/pdf/create I'm freaking out ... I'll pass this to the hosting and see if they give me solution that 150 euros a year is not a joke, at least for my pocket ...

Thanks again!!! A hug!
